srtcore/channel.cpp

```diff
@@ -525,8 +577,28 @@ int CChannel::sendto(const sockaddr* addr, CPacket& packet) const
       mh.msg_namelen = m_iSockAddrSize;
       mh.msg_iov = (iovec*)packet.m_PacketVector;
       mh.msg_iovlen = 2;
-      mh.msg_control = NULL;
-      mh.msg_controllen = 0;
+      bool have_set_src = false;
+
+#ifdef SRT_ENABLE_PKTINFO
+      if (m_bBindMasked && !source_addr.isany())
+      {
+          if (!setSourceAddress(mh, source_addr))
```

Comment:
Nothing relevant to this PR, but the alignment of the whole block is wrong.
Starting from line 574 `#ifndef _WIN32`.
Would be good to fix it. Even though clang format is planned for some distant future.

Reply:
Yeah, this is so since UDT, the extra indent is due to `#ifdef _WIN32`. This should be taken care of when we perform the "great reformation" with `clang-format`.